node_modules/oae-tenants/lib/api.js

```diff
@@ -72,6 +73,24 @@ Pubsub.on('oae-tenants', function (message) {
     });
 });
 
+/*!
+ * Create the guest tenant if it doesn't exist already. This needs to happen when the system
+ * is ready, are there are a lot of modules that need to be initialized who listen for `create-tenant` events.
+ */
+OaeEmitter.on('ready', function() {
+    // If the guests tenant already exists, we can return here
+    if (getTenant(serverConfig.guestsTenantAlias)) {
+        return;
+    }
+
+    // Create the guests tenant
+    _createTenant(serverConfig.guestsTenantAlias, 'Guests tenant', serverConfig.guestsTenantHost, function(err) {
```

Comment:
This reminds me that users in the "Guests tenant" will just see `Guests tenant` as the loggedin visibility option, which is almost definitely going to be odd. We could remove the `loggedin` option on a guest tenant, but I'm not sure if that helps.